index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://kit.fontawesome.com/b957cbbe16.js" crossorigin="anonymous"></script>
+
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="mobile-container">
+    
+    <nav>
+    <div> 
+      <p class="viewmobile">Welcome
+        <i class="fa-solid fa-bars"></i>
+      </p>
+     
+    </div>
+     
+    </nav>
+
+
+  </header>
+  
+    <main class="info">
+      <h2>Hey there. I’m <br> Mandela <br>
+ <span> I’m a software developer </span>
+</h2>
+  <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hesitate to contact me.
+  </p>
+
+</main>
+
+  <div class="socialmedia">
+    <a target="_blank" href="#">
+    <img src="images/twitter.svg" alt=" twitter handles">
+  </a>
+
+  <a target="_blank" href="#">
+    <img src="images/Linkedinicon.svg" alt=" linked handles">
+  </a>
+
+  <a target="_blank" href="#">
+    <img src="images/github.svg" alt=" github handles">
+  </a>
+  
+  </div>
+
+</body>
+</html>
```

Comment:
Kindly change the PR description to be more descriptive like mentioning what you did exactly in this PR like:
- Add navbar. 
- Add hero section.